openage/convert/interface/interfacerename.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+""" Renaming interface assets and splitting into direvtories """
+
+from .interfacehardcoded import ingame_hud_background_index
+
+
+def interface_rename(filename):
+    """
+    Returns a human-usable name according to the original and hardcoded metadata.
+    """
+    try:
+        return "interface/hud" + str(ingame_hud_background_index(filename)).zfill(4) + ".slp"
+    except ValueError:
+        return filename
```

Comment:
The string formatting thingy doesn't matter much, I was just trying to get rid of the exception handling. But it you think your current way is the best one, then it's ok.